es_doxygen_apache.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Notice: can't find usage of this script in cms jenkins

Reply:
right, this was also used by doxygen server which has been deprecated. Now we server cmssw doxygen via CERN IT web infrastructure